src/test/ui/issues/issue-73229.rs

```diff
@@ -0,0 +1,33 @@
+// check-pass
```

Comment:
I just noticed that this code hangs when run, though it compiles fine. Is that intentional?

Reply:
Yeah that is expected, it's because of the `loop {}`

Reply:
`check-pass` does not run the tests. It doesn't even codegen it, only checks that an equivalent of `cargo check` succeeds.

Reply:
Ah, somehow I missed the loop. Thanks!